<!-- Greg Attra -->
<!-- 03/28/2021 -->

<launch>
    <rosparam file="$(find roborex)/config/params.yaml" command="load"/>
    <node name="explore_behavior" pkg="roborex" type="roborex_explore_behavior" output="screen"/>
    <node name="tcp_server" pkg="roborex" type="tcp_endpoint.py" output="screen"/>
</launch>